src/services/codefixes/fixAddMissingMember.ts

```diff
@@ -89,9 +89,8 @@ namespace ts.codefix {
             if (!(symbol && leftExpressionType.flags & TypeFlags.Object && symbol.flags & SymbolFlags.Class)) {
                 return undefined;
             }
-            const classDeclaration = cast(first(symbol.declarations), isClassLike);

```

Comment:
should we check that we found a class?

Reply:
I think it should be guaranteed to exist if the symbol has `SymbolFlags.Class`.